dev-docs/bidders/pixfuture.md

```diff
@@ -10,12 +10,121 @@ prebid_member: false
+
+---
+
+### Features
+
+| Feature                  | Status                |
+|--------------------------|-----------------------|
+| **Bidder Code**          | `pixfuture`           |
+| **Prebid.org Member**    | No                    |
+| **Prebid.js Adapter**    | Yes                   |
+| **Prebid Server Adapter**| Yes                   |
+| **Media Types**          | Display (Banner)      |
+| **Multi Format Support** | Contact support       |
+| **TCF-EU Support**       | Yes                   |
+| **IAB GVL ID**           | 839                   |
+| **GPP Support**          | Contact support       |
+| **DSA Support**          | Contact support       |
+| **USP/CCPA Support**     | Yes                   |
+| **COPPA Support**        | Yes                   |
+| **Supply Chain Support** | Contact support       |
+| **Demand Chain Support** | Contact support       |
+| **Safeframes OK**        | Contact support       |
+| **Supports Deals**       | Contact support       |
+| **Floors Module Support**| Contact support       |
+| **First Party Data Support** | Contact support   |
+| **User IDs**             | `criteoId`, `unifiedId`, `id5Id`, `sharedId`, `identityLink`, `liveIntentId`, `fabrickId` |
+| **ORTB Blocking Support**| Contact support       |
+| **Privacy Sandbox**      | Contact support       |
+
+For features marked "Contact support," reach out to <support@pixfuture.com> for more information.
+
+---
+
+### Bid Parameters
+
+| Name       | Scope    | Description                                                        | Example       | Type     |
+|------------|----------|--------------------------------------------------------------------|---------------|----------|
+| `pix_id`   | required | A unique ID for your site’s ad placement. Corresponds to each ad size. | `"12312345"`    | `string` |
+
+The `pix_id` is a unique identifier provided by PixFuture and must be specified for each ad placement/size combination. This parameter is consistent across both Prebid.js and Prebid Server.
+
+---
+
+### Prebid.js Integration
+
+#### Basic Prebid.js Example
+
+Below is an example of configuring ad units for PixFuture in Prebid.js:
+
+```javascript
+var adUnits = [
+    {
+        code: 'test-div-300x250',
+        mediaTypes: {
+            banner: {
+                sizes: [[300, 250]]
+            }
+        },
+        bids: [
+            {
+                bidder: 'pixfuture',
+                params: {
+                    pix_id: "11234567890"
+                }
+            }
+        ]
+    },
+    {
+        code: 'test-div2-728x90',
+        mediaTypes: {
+            banner: {
+                sizes: [[728, 90]]
+            }
+        },
+        bids: [
+            {
+                bidder: 'pixfuture',
+                params: {
+                    pix_id: "0987654321"
+                }
+            }
+        ]
+    }
+];
+
+## Additional "Send All Bids" Ad Server Keys
```

Comment:
Also remove this section -- it's automatically added by the docs system.

Reply:
The entire "Send All Bids" Ad Server Keys section.

Reply:
Fixed. Both unnecessary sections have been removed.